types/k6/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/k6",
-    "version": "0.57.9999",
+    "version": "1.0.9999",
```

Comment:
I guess we can't have an rc  here?

Reply:
@mstoykov yep, that's my interpretation of https://github.com/grafana/k6-DefinitelyTyped/tree/master?tab=readme-ov-file#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library. Or at least is a safe play.

Now the question is, should we release this `types` update now and interpret all the upcoming RCs and the final as patches. Or should we publish only when we release a final version? I would prefer to do the latter but I'm open to accept alternatives.